spacy/errors.py

```diff
@@ -887,6 +887,7 @@ class Errors(metaclass=ErrorsWithCodes):
     E1021 = ("`pos` value \"{pp}\" is not a valid Universal Dependencies tag. "
              "Non-UD tags should use the `tag` property.")
     E1022 = ("Words must be of type str or int, but input is of type '{wtype}'")
+    E1023 = ("Couldn't read EntityRuler from the {path}. This {frmt} file doesn't exist.")
```

Comment:
I think this parameter can just be `format`, it's short enough not to be abbreviated :-)

Reply:
Heh ok:) , I killed this param already.